### DISCUSSION TITLE
Reordering messages

### DISCUSSION THREAD

**Discussion by deandonaldson** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#171] [Feature Requests & Ideas]
Copy and paste is a life saver, but the ability to move items up and down the stack would be great.

Drag/drop would be amazing but even arrows next to each line item in the web editor to move them up and down would be awesome.

**Comment by Glennmack** [comment]
Agreed - very useful features!

**Comment by simonaglover95** [comment]
done!
